wormchain/contracts/tools/.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
can we add a `.nvmrc` file as well to pin the node version?

Reply:
This falls under the same node version requirements as the rest of the monorepo. I don't think it makes sense to add one here but no where else.